CHANGELOG.md

```diff
@@ -10,22 +10,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [12.2.0] - 2022-10-25
 
-### Fixed
-- Fixed problem related to calculation of editor's width [#3815](https://github.com/handsontable/handsontable/issues/3815)
-- Fixed problem with overwriting source data incorrectly when there are some merged cells [#6359](https://github.com/handsontable/handsontable/issues/6359)
-- Fixed problems related to placing instance in some flex containers and setting an  overflow property for some of them to hidden, scroll or auto value. [#9545](https://github.com/handsontable/handsontable/issues/9545)
-- Fixed an issue with being unable to effectively change the `type` meta multiple times. [#9734](https://github.com/handsontable/handsontable/issues/9734)
-- Fixed an issue with dropdownMenu not being displayed on a nestedHeaders-enabled instance with all rows trimmed out. [#9753](https://github.com/handsontable/handsontable/issues/9753)
-- Fixed an issue where release script exits with an error code. [#9941](https://github.com/handsontable/handsontable/issues/9941)
+For more information on this release, see:
+- [Release notes](https://handsontable.com/docs/release-notes/#_12-2-0)
+- [Blog post](https://handsontable.com/blog/articles/handsontable-12.2.0)
 
 ### Added
-- Added a new feature that lets you add rows and columns in a specified position (above or below the given index). [#9471](https://github.com/handsontable/handsontable/issues/9471)
+- Added a new feature that lets you add rows and columns in a specified position (above or below a specified index). [#9471](https://github.com/handsontable/handsontable/issues/9471)
```

Comment:
As a rule of thumb, I think that the names of the new, changed or deprecated APIs should always be explicitly listed in the release notes for clarity and searchability.

Reply:
thanks!